addons/account/views/account_reconcile_model_views.xml

```diff
@@ -186,11 +190,14 @@
 
                                             <field name="sequence"
                                                    widget="handle"/>
-                                            <field name="account_id"/>
+                                            <field name="account_id"
+                                                   required="(parent.counterpart_type == 'general' and parent.rule_type == 'writeoff_button')
```

Comment:
If I understand it right, we don’t want to force any accounts here because when the move is created, the default accounts for moves will be used (for example, for invoices, the account Product Sales will be used by default). The user can still change the account for the product move line before posting the move.

But, this showed me that I was overlooking the scenario when they do set an account. Now, if an account is set, it is being used in the move creation.